src/games/strategy/triplea/ui/MapPanel.java

```diff
@@ -624,7 +624,7 @@ private void updateUndrawnTiles(final List<Tile> undrawnTiles, final int preDraw
         if (tile.isDirty()) {
           undrawnTiles.add(tile);
         } else if (forceInMemory) {
-          images.add(tile.getRawImage());
+          images.add(tile);
```

Comment:
This List had no Type safety... Image != Tile both of those types were added to this list at the same time... Since we can get this Image from the tile I added the tile instead...
I have no idea why this list is even needed because the values of this list are never gotten only set and cleared